test/services/permissions.test.js

```diff
@@ -9,154 +9,164 @@ describe('services > permissions', () => {
   const { forestUrl } = context.inject();
   const nockObj = nock(forestUrl);
 
-  describe('check permissions', () => {
-    describe('with some good permissions data on rendering 1', () => {
-      describe('with the "list" permission', () => {
-        it('should return a resolved promise', async () => {
-          expect.assertions(1);
+  describe('with teamsACL permissions format', () => {
+    describe('check permissions', () => {
+      describe('with some good permissions data on rendering 1', () => {
+        describe('with the "list" permission', () => {
+          it('should return a resolved promise', async () => {
+            expect.assertions(1);
 
-          PermissionsChecker.resetExpiration(1);
-          PermissionsChecker.cleanCache();
-          nock.cleanAll();
-          nockObj.get('/liana/v2/permissions?renderingId=1')
-            .reply(200, {
-              Users: {
-                collection: {
-                  list: true,
+            PermissionsChecker.resetExpiration(1);
+            PermissionsChecker.cleanCache();
+            nock.cleanAll();
+            nockObj.get('/liana/v3/permissions?renderingId=1')
+              .reply(200, {
+                meta: { rolesACLActivated: false },
+                data: {
+                  Users: {
+                    collection: {
+                      list: true,
+                    },
+                  },
                 },
-              },
-            });
+              });
 
-          await new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'list')
-            .then(() => { expect(true).toStrictEqual(true); });
+            await new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 })
+              .then(() => { expect(true).toStrictEqual(true); });
```

Comment:
You are using `await` so get rid of the `then` it will be much easier to read

Reply:
This is old code but you're right